acknowledgements.md

```diff
@@ -40,17 +40,17 @@ Really useful tool for benchmarking .NET code! Available for use under MIT Licen
 
 Even though they are no longer directly used for NSubstitute, the following projects were really helpful in building previous NSubstitute versions.
 
-## Microsoft Reactive Extensions for .NET (Rx) [http://msdn.microsoft.com/en-us/devlabs/ee794896]
-Used to provide .NET 3.5 with some of the neat concurrency helper classes that ship with out of the box with .NET 4.0. Distributed under a EULA [http://msdn.microsoft.com/en-us/devlabs/ff394099]. No longer required since we stopped supporting .NET 3.5.
+## Microsoft Reactive Extensions for .NET (Rx) [https://msdn.microsoft.com/en-us/devlabs/ee794896]
+Used to provide .NET 3.5 with some of the neat concurrency helper classes that ship with out of the box with .NET 4.0. Distributed under a EULA [https://msdn.microsoft.com/en-us/devlabs/ff394099]. No longer required since we stopped supporting .NET 3.5.
 
-## Microsoft Ilmerge [http://research.microsoft.com/en-us/people/mbarnett/ilmerge.aspx]
+## Microsoft Ilmerge [https://research.microsoft.com/en-us/people/mbarnett/ilmerge.aspx]
```

Comment:
Link redirects, may https://github.com/dotnet/ILMerge now?

Reply:
👍

Reply:
@dtchepak Should we still keep it here if we don't use the tool anymore? 😉

Reply:
@zvirja I'm in two minds about this. It was very helpful for a long time. :) And NuGet still has versions published that have ilmerged artefacts. 🤔

Reply:
@dtchepak Let's keep it then - it worth nothing to keep it. Let's help project to be scored slightly higher by search engines 😄

Reply:
changed to  https://github.com/dotnet/ILMerge